frontend/app/helpers/path-helper.js

```diff
@@ -217,26 +217,6 @@ module.exports = function() {
       return PathHelper.apiWorkPackagesPath() + '/column_sums';
     },
 
```

Comment:
@NobodysNightmare you should also remove https://github.com/NobodysNightmare/openproject/blob/feature/remove_apiv2_access/frontend/app/helpers/path-helper.js#L32